module4/projects/tech_challenges/index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Technical Challenges
+layout: page
+---
+
+Technical challenges may be given as part of the interview process to help assess candidates on how they break down problems, communicate their logic and reasoning, demonstrate their technical knowledge, and implement a working solution. Our goal is to provide you with the opportunity to practice these skills and develop a good process. Over the course of 1 week, students will complete daily timed technical challenges and reflect on their approach to these problems.
+
+These scenarios are focused on helping you develop a good approach and do not always allow enough time to complete the challenge. Do not be discouraged if you are unable to implement a solution in the alotted time. Some of the challenges are very difficult and assigned intentionally to emphasize process over results.
+
+<section class="call-to-action">
+
+### Technical Prompts
+
+We will use these [Technical Prompts](./prompts) for our practice throughout Mod 4.
+
+</section>
+
+## Technical Challenge Format
+
+Technical challenges will either be solo or paired.
+
+After each challenge, students should track their reflections in a [GitHub Gist](https://gist.github.com/) that they will submit at the end of the week.
```

Comment:
Similar to the survey, this is not something I've ever asked for or reviewed given just capacity limitations. I like the idea of reflections. Perhaps we should just chop off the part about submitting the gist? OR does this feel important enough to require and at least validate is being submitted? If so, we can definitely prioritize. I'd love to hear your thoughts on this. I think I'm too deep into Mod 4 to objectively assess that right now.

Reply:
I've removed the note about submitted and adjusted to allow them to use whatever note-taking tool they prefer.  I dont think this requires submission.